src/events/common.ts

```diff
@@ -1,16 +1,45 @@
 /* eslint-disable camelcase */
+import { codeBlock } from "discord.js";
 import type { ArgsOf, Client } from "discordx";
 import { Discord, On } from "discordx";
 
-import sendCharProfile from "../lib/discord/Character/helpers/sendCharProfile";
+import CharacterPostEmbed from "../lib/discord/Character/classes/CharacterPostEmbed";
+import CharacterFetcher from "../lib/pocketbase/CharacterFetcher";
 
 @Discord()
 export class Example {
   @On()
-  messageCreate([message]: ArgsOf<"messageCreate">, _client: Client): void {
-    if (!message.content.startsWith("!test")) {
-      return;
+  async messageCreate([message]: ArgsOf<"messageCreate">, _client: Client): Promise<void> {
+    if (message.content.startsWith("gwynevere:")) {
+      const characterFetcher = new CharacterFetcher();
+      const characterData = await characterFetcher.getCharactersByUserId({
+        page: 1,
+        userId: message.author.id,
+      });
+      const characterEmbed = new CharacterPostEmbed(characterData.items[0]);
+
+      message.channel.send(
+        await characterEmbed.createMessageOptions({
+          attachmentUrl: message.attachments.first()?.url,
+          content: message.content,
+          to: "message",
+        })
+      );
+    }
+    if (
+      message.author.id === message.guild?.ownerId ||
+      message.author.id === "777624354560802876"
+    ) {
+      if (message.content.startsWith("```js")) {
```

Comment:
This is really dangerous, do we really need that?

Reply:
If the response is "Yes" you can just close this comment, else we might not want this on our App code

Reply:
Yeah. It's only going to be used by our user Ids, so unless you want to fs.unlink() everything I think we're fine